### ISSUE TITLE
Mines - guessing required

### ISSUE THREAD

**Issue by RandomNameTag** [chrisboyle/sgtpuzzles#674]
**Describe the bug**
Mines game is supposed to be set so that no guessing is required. In the game i played, i reached a point where guessing is required (see printscreen)

**To Reproduce**
n.a

**Expected behavior**
Sufficient clues should be provided so that no guessing is required to progress

**Version info (optional, but sometimes helpful)**
 Version 2024-03-29-2245-fd304c53

![Screenshot_20241129_092615_Puzzles](https://github.com/user-attachments/assets/49c4cadc-cf81-4219-a1f8-909ea0587aea)

**Comment by doopl** [comment]
You can use the fact that there are 8 safe squares remaining to proceed with the puzzle. If you like I will explain how.

**Comment by chrisboyle** [comment]
The above explanation looks good to me 👍

**Comment by RandomNameTag** [comment]
Right! Thanks a lot for the explanation, I guess i never got to think that deep!
Also, kuddos I absolutely love the app.
